doc/GUIDE.md

```diff
@@ -231,6 +231,7 @@ public static MethodHandle foo$handle() { ... } // 4
 First and foremost, there is a static wrapper method that is generated that can be used to
 call the C function (1). Besides that, there are also several accessors that return
 additional meta-data for the method:
+
```

Comment:
You could also add an item for (1) instead of having separate text above?

Reply:
Done

Reply:
Now veering into nitpicks: but I note that we have an asymmetry between structs and functions. In functions we try to call out the function accessor (probably on the basis that this is the symbol the user is likely to be looking for). But for structs we just list all the members one after the other (even though the most likely is probably (1) - e.g. getter/setter). I think the document could be improved (optionally) by rectifying this one way or the other.